CHANGELOG.md

```diff
@@ -4,6 +4,30 @@ All notable changes to this project will be documented in this file.
 The changelog format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 
+## [1.0.9][v1.0.9] - 29-Nov-2021
+### Milestone: Stabilize and bug fix
+
+#### Added
+- Added open api client integration[#1776](https://github.com/symbol/desktop-wallet/pull/1776)
+- Added Korean language [#1794](https://github.com/symbol/desktop-wallet/pull/1794)
+- Show update warning when users are on old versions[#1393](https://github.com/symbol/desktop-wallet/issues/1393)
+
+#### Changed
+- Used openapi client for statistics-service API calls [#1777](https://github.com/symbol/desktop-wallet/pull/1777)
+- Updated symbol-sdk version to 1.0.3 [#1756](https://github.com/symbol/desktop-wallet/pull/1756)
+- Replaced old explorer url with the new one [#1766](https://github.com/symbol/desktop-wallet/pull/1766)
+
```

Comment:
```suggestion
Harvesting page improvements: find node by `nodePublicKey` from statistics service instead of local storage, multi-device operation support [#1781](https://github.com/symbol/desktop-wallet/pull/1781)
```